src/test/java/core/basesyntax/service/impl/BalanceOperationTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.FruitStorage;
+import core.basesyntax.storage.Storage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationTest {
+    private FruitStorage storage;
+    private BalanceOperation balanceOperation;
+
+    @BeforeEach
+    public void setUp() {
+        storage = new Storage();
+        balanceOperation = new BalanceOperation(storage);
+    }
+
+    @Test
+    public void handle_UpdateFruitQuantity_WhenFruitDoesNotExist() {
+        FruitTransaction transaction = new FruitTransaction();
+        transaction.setFruit("apple");
```

Comment:
create constants for fruits and quantities

Reply:
In all test classes, there are no constants. Should I add constants only in the classes where you left a comment, or should I add them in all test classes?